### DISCUSSION TITLE
Offline Mode is Not Working Properly

### DISCUSSION THREAD

**Discussion by afsalakkarammal** [KRTirtho/spotube#1295] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I've downloaded some songs . Then i went to offline mode . But i cannot find the songs in Downloads

### Expected Behavior

Songs i downladed should be present in Downloads while im offline

### Steps to reproduce

1. i open the app
2. downloaded some songs 
3. went offline 
4. went to Downloads
5. Showing "Currently Downloading(0)"
![WhatsApp Image 2024-03-05 at 01 04 36_e83a20e0](https://github.com/KRTirtho/spotube/assets/125533278/7a9239b3-bf40-45f9-8902-93989d1119ca)

### Operating System

Android

### Spotube version

v3.4.1

### Installation source

Play Store (Android)

### Additional information

Need to improve Cache Handling 

### Self grab

- [X] I'm ready to work on this issue!

**Comment by KRTirtho** [comment]
Go to the "Tracks" tab :)
